js/cpu6502.ts

```diff
@@ -991,77 +997,94 @@ export default class CPU6502 {
         return unk;
     }
 
-    private dumpArgs(addr: word, m: Mode, symbols: symbols) {
-        function toHexOrSymbol(v: word, n?: number) {
+    private dumpRawOp(parts: byte[]) {
```

Comment:
Should this all be moved to `debugger.ts`?

Reply:
I thought about it but left it here to allow it to use some of the opcode tables, but it could be moved.